specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/examples/CreateAManagedDiskFromAPlatformImage.json

```diff
@@ -39,12 +46,18 @@
         "name": "myDisk",
         "location": "West US",
         "properties": {
-          "provisioningState": "Updating",
```

Comment:
is promotionCode missing?

Reply:
its optional, some purchase plan have it and some dont, I am not able to find the one where its present.